### PR TITLE
docs: Updated for Makefile added

### PR DESCRIPTION
**Pull request by vli11**
Closes: issue #58

_The PR review is to check for sustainability and correctness.  Sustainability is actually more business critical as correctness is largely tested into the code over time.   Its useful to keep in mind that SW often outlives the HW it was written for and engineers move from job to job so it is critical that code developed for Intel be supportable across many years.  It is up to the submitter and reviewer to look at the code from a perspective of what if we have to debug this 3 years from now after the author is no longer available and defect databases have been lost.  Yes, that happens all the time when we are working with time scales of more than 2 years.  When reviewing your code it is important to look at it from this perspective._

Author Mandatory (to be filled by PR Author/Submitter)
------------------------------------------------------
- Developer who submits the Pull Request for merge is required to mark the checklist below as applicable for the PR changes submitted.  
- Those checklist items which are not marked are considered as not applicable for the PR change.  
- Items marked with an asterisk suffix are mandatory items to check and if not marked will be treated as non-compliant pull requests by the developers for Inner Source Development Model (ISDM) compliance

### PULL DESCRIPTION
_Provide a 1-2 line brief overview of the changes submitted through the Pull Request..._

### REFERENCES
Reference URL for issue tracking (Github): **\<URL to be filled>**

- [x] Added label to the Pull Request for easier discoverability and search

### CODE MAINTAINABILITY
- [x] **_Commit Message meets guidelines as indicated in the URL\*_**
	- https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md
- [x] **_Every commit is a single defect fix and does not mix feature addition or changes\*_**
- [ ] Added required new tests relevant to the changes
	- [ ] PR contains URL links to functional tests executed with the new tests 
- [x] Updated Documentation as relevant to the changes
- [ ] Updated Build steps/commands changes as relevant
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide description)
- [ ] Specific instructions or information for code reviewers (If any):

Maintainer Mandatory (to be filled by PR Reviewer/Approving Maintainer)
-----------------------------------------------------------------------
- Maintainer who approve the Pull Request for merge are required to mark the checklist below as appropriate for the PR change reviewed as key proof of attestation indicating reasons for merge. 
- Those checklist items which are not marked are considered as not applicable for the PR change. 
- Items marked with an asterisk suffix are mandatory items to check.

### QUALITY CHECKS
- [ ] Architectural and Design Fit
- [ ] **_Quality of code (At least one should be checked as applicable)\*_**
	- [ ] Commit Message meets guidelines
	- [ ] PR changes adhere to industry practices and standards
	- [ ] Adopted domain specific coding standards ([SBL Coding Standards](https://github.com/tianocore-docs/Docs/raw/master/Specifications/CCS_2_1_Draft.pdf))
	- [ ] Error and exception code paths implemented correctly
	- [ ] Code reviewed for domain or language specific anti-patterns
	- [ ] Code is adequately commented
	- [ ] Code copyright is correct
	- [ ] Tracing output are minimized and logic
	- [ ] Confusing logic is explained in comments
	- [ ] Commit comment can be used to design a new test case for the changes
- [ ] **_Test coverage shows adequate coverage with required CI functional tests pass on all supported platforms\*_**
- [ ] **_Static code scan report shows zero critical issues\*_**

### CODE REVIEW IMPACT
- Summary of Defects Detected in Code Review: **\<%P1*xx,P2*xx,P3*xx,P4*xx%>** \
Note P1/P2/P3/P4 denotes severity of defects found (Showstopper/High/Medium/Low) and xx denotes number of defects found

# _Code must act as a teacher for future developers_

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,10 +6,10 @@
 build-all: build-soc build-dgpu
 
 build-soc:
-	./docker-build.sh soc
+	./docker-build.sh soc $(PROXY)
```
Comment (jim-wang-intel) [inline Makefile:9]: this doesn't work- as you have both http_Proxy and https_Proxy and they are different

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]

**Comment by brian-intel** [comment]
Can you also update docs_src/release-notes/v1-0-0.md "Vision Self Checkout" -> "Automated Self Checkout"
and
README.md "Vision Self Checkout" -> "Automated Self Checkout"

File: Makefile (hunk 1 above)
Comment (vli11) [reply Makefile:9]: I have tested this before PR, so it is working as it is. current change is reflecting what was there on the doc as input for proxy.

**Comment by vli11** [comment]
> Can you also update docs_src/release-notes/v1-0-0.md "Vision Self Checkout" -> "Automated Self Checkout" and README.md "Vision Self Checkout" -> "Automated Self Checkout"

I will make a separated PR for overall name change

File: docs_src/pipelinesetup.md [hunk 2]
```diff
@@ -46,7 +48,7 @@ You must build the provided component services and create local docker images. D
 !!! build with proxy information:
     If docker build system requires a proxy network, please provide the proxy URL after the first argument.  For example, to build the reference design docker image with the proxy information:
 ```bash
-./docker-build.sh <soc|dgpu> http://http_proxy_server_ip:http_proxy_server_port http(s)://https_proxy_server_ip:https_proxy_server_port
+make build-all PROXY="http://http_proxy_server_ip:http_proxy_server_port https://https_proxy_server_ip:https_proxy_server_port"
```
Comment (jim-wang-intel) [inline docs_src/pipelinesetup.md:51]: need to update this

File: docs_src/pipelinerun.md [hunk 3]
```diff
@@ -178,7 +178,7 @@ The output in results/pipeline0.log file lists FPS (frames per second) during pi
 ...
 ```
 
-The results/ directory is volume mounted to the pipeline container, the log files inside the results/ directory will keep on growing as the pipeline is still running, you can call ./stop_all_docker_containers.sh to stop the pipeline and all running containers.
+The automated-self-checkout/results/ directory is volume mounted to the pipeline container, the log files inside the automated-self-checkout/results/ directory will keep on growing as the pipeline is still running, you can call `make clean-all` to stop the pipeline and all running containers.
```
Comment (jim-wang-intel) [inline docs_src/pipelinerun.md:181]: could you also mention about `make clean` and `make clean-simulator`?

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]
some improvement on docs

File: docs_src/pipelinerun.md [hunk 4]
```diff
@@ -27,7 +27,13 @@ You have to get your choices for #1, #2, #3 above to start the pipeline run, see
...
 
 ### Stop pipeline run
-You can call ./stop_all_docker_containers.sh to stop the pipeline and all running containers, hence the results directory log files will stop growing.
+You can call `make clean-all` to stop the pipeline and all running containers, hence the results directory log files will stop growing. Below is table of commands you can call to clean things up per your needs:
+
+| Clean Containers Options   | Command                         |
+| ---------------------------| --------------------------------|
+| clean simulator containers | <pre>make clean-simulator</pre> |
+| clean sco-* containers     | <pre>make clean</pre>           |
+| clean all containers       | <pre>make clean-all</pre>       |
```
Comment (jim-wang-intel) [inline docs_src/pipelinerun.md:36]:
```suggestion
| clean both simulator and self-checkout containers       | <pre>make clean-all</pre>       |
```

File: docs_src/pipelinerun.md [hunk 5]
```diff
@@ -27,7 +27,13 @@ You have to get your choices for #1, #2, #3 above to start the pipeline run, see
 Once pipeline run has started, you will expect containers to be running, see [check for pipeline run success](#check-for-pipeline-run-success); For a successful run, you should expect results/ directory filled with log files and you can watch these log files grow, see [sample output log files](#sample-output).
 
 ### Stop pipeline run
-You can call ./stop_all_docker_containers.sh to stop the pipeline and all running containers, hence the results directory log files will stop growing.
+You can call `make clean-all` to stop the pipeline and all running containers, hence the results directory log files will stop growing. Below is table of commands you can call to clean things up per your needs:
```
Comment (jim-wang-intel) [inline docs_src/pipelinerun.md:30]:
```suggestion
You can call `make clean-all` to stop the pipeline and all running containers, hence the results directory log files will stop growing. Below is the table of make commands you can call to clean things up per your needs:
```

**Review by jim-wang-intel — APPROVED** [review]
LGTM, good job 👍

**Review by brian-intel — APPROVED** [review]

**Pull request merged**